components/cards/cards.css

```diff
@@ -0,0 +1,151 @@
+.card-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 1rem;
+    background-color: var(--grey);
+    padding: 1rem;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+    position: relative;
+}
+
+
+.card {
+    display: flex;
+    flex-direction: column;
+    max-width: 20rem;
+    margin: 1rem;
+    border-radius: 2px;
+    background-color: white;
+    border-radius: 5px;
+
+}
+
+.card-img {
+    width: 100%;
+    height: auto;
+    background-size: cover;
+    border-radius: 5px 5px 0 0;
+}
+
+.card-info {
+    font-weight: 600;
+    margin-bottom: 0.5rem;
+}
+
+.card-body {
+    padding: 0.75rem 0.75rem 0;
+}
+
+.card-btn {
+    padding: 0.75rem;
+}
+
+.card-btn button {
+    margin-top: 0.5rem;
+    width: 100%;
+}
+
+.card-header {
+    position: relative;
+}
+
+.card-badge {
+    position: absolute;
+    top: 1rem;
+    left: 0;
+    padding: 0.5rem;
+    background-color: #fb923c;
```

Comment:
use variables

Reply:
I will Mimansha , thankyou for pointing it out :)